email_templates/cashEmail/text.handlebars

```diff
@@ -0,0 +1,7 @@
+{{listing_name}} has been purchased!
+
+IOU: By purchasing {{listing_name}}, {{listing_buyer}} has agreed to pay the 
+amount of ${{listing_price}} in cash to {{listing_creator}}.
+
+Please save this email for your records until the IOU has been resolved. 
+Thank you for using Obay!
```

Comment:
these placeholders are definitely fine for now. I can take care of ensuring that everything you want inserted is inserted when the sendCashEmail function is called, if you want to focus on styling.